assets/icons/chatbot-bot.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="400" height="400" viewBox="0 0 400 400" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M399.933 56.72V343.387C399.912 358.413 393.927 372.817 383.293 383.433C372.66 394.048 358.247 400.007 343.223 400H56.6572C41.633 399.993 27.2261 394.02 16.6023 383.395C5.97856 372.769 0.00706526 358.36 0 343.333V56.6667C0.00706526 41.6399 5.97856 27.2306 16.6023 16.6051C27.2261 5.97956 41.633 0.00706643 56.6572 0H343.276C358.309 0.010605 372.721 5.99155 383.346 16.6278C393.971 27.2641 399.937 41.6851 399.933 56.72Z" fill="#FBDA16"/>
```

Comment:
:wrench: At the moment we only plan the chat for integreat, but won't hurt to make the icons configurable already for other build configs.
```suggestion
<path d="M399.933 56.72V343.387C399.912 358.413 393.927 372.817 383.293 383.433C372.66 394.048 358.247 400.007 343.223 400H56.6572C41.633 399.993 27.2261 394.02 16.6023 383.395C5.97856 372.769 0.00706526 358.36 0 343.333V56.6667C0.00706526 41.6399 5.97856 27.2306 16.6023 16.6051C27.2261 5.97956 41.633 0.00706643 56.6572 0H343.276C358.309 0.010605 372.721 5.99155 383.346 16.6278C393.971 27.2641 399.937 41.6851 399.933 56.72Z" fill="currentColor"/>
```